phpcs.xml.dist

```diff
@@ -20,6 +20,7 @@
         <exclude name="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint"/> <!-- we can do it in doctrine-bundle 3.0 -->
         <exclude name="SlevomatCodingStandard.Classes.SuperfluousInterfaceNaming.SuperfluousSuffix"/>
         <exclude name="SlevomatCodingStandard.Classes.SuperfluousAbstractClassNaming.SuperfluousPrefix"/>
+        <exclude name="SlevomatCodingStandard.Classes.ModernClassNameReference.ClassNameReferencedViaFunctionCall" /> <!-- PHP 8.0 required -->
```

Comment:
this doesn't look right. We run phpcs under php 7.4

Reply:
I've reverted the line. However, that rule is a pitfall for everyone who uses PHP 8 locally. 🤔

Reply:
I see. In that case you could leave it in. But I think we also have similar issues with psalm, it will report different issues on different php versions, not much can be done about it.

Reply:
I created #1400 to update the PHPCS job to PHP 8. That job should fail without that line.